src/Components/Members/MemberEditCreate.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { CKEditor } from "@ckeditor/ckeditor5-react";
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+import Swal from "sweetalert2";
+import "../FormStyles.css";
+
+const MemberEditCreate = () => {
+  const [member, setMember] = useState({
+    name: "",
+    description: "",
+    imageMember: "",
+    social_media: "",
+  });
+
+  const handleChange = (e) => {
+    if (e.target.name === "name") {
+      setMember({ ...member, name: e.target.value });
+    }
+
+    if (e.target.name === "image") {
+      const fileValid =
+        e.target.files[0].type === "image/jpg" ||
+        e.target.files[0].type === "image/png";
+      if (fileValid) {
+        setMember({ ...member, imageMember: e.target.value });
+      } else {
+        Swal.fire({
```

Comment:
On Utils you can add a file alerts and a function on that file called showErrorAlert
```
const showErrorAlert = (message, title?) => {
    Swal.fire({
          icon: "error",
          title: title || "Oops...",
          text: message,
    })
}
```
and then you call it here:
```
showErrorAlert ("Please select a valid image type jpg or png");
```

Reply:
changes added